node/expressjs/middlewares-in-express-js.md

```diff
@@ -0,0 +1,61 @@
+## Middlewares in Express JS
+
+It's been a long time that NODE JS has become a part of JS Ecosystem. Any JS developer Front End or Full Stack would be familiar with Node JS. Any one who has worked on Node would have also worked with Express JS.
+
+So, Express JS is nothing but a framework to help organize your web application into an MVC architecture on the server side. It basically helps you manage everything from routes to handling requests and views for your web application.
+
+### What are Middlewares
+
+A middleware is just a function which takes request(req), response(res) and the next middleware function(next) as the arguments. These are the function which helps to intercept the incoming request, perform a specific task and pass the execution to next middleware or route handler or even they can end the request-response cycle by returning a response.
+
+![what are middleware](https://qph.fs.quoracdn.net/main-qimg-78c4c87d1e7f41795801b4a49b721d6f)
+
+### How Middleware Works
+
+Any request on the Node Server will be accessed by all the middlewares in the request-response cycle in a sequential order. The line of order in which the middlewares are defined matters a lot.You can take it as that the Output of first middleware is Input of the second middleware for request and response object. A middleware take a 3rd argument which is the next middleware or the route function. If your middleware doesn't returns a response it should call next() to pass the control to the next middleware function otherwise the request will always be hanging in the application.
```

Comment:
Instead of capitalizing words like Output, you should keep them lowercase but make them bold instead **output**